create_pdfs.py

```diff
@@ -37,8 +37,9 @@ def parse_args(args=sys.argv[1:]):
     parser.add_argument('-i', '--issued-date', help='optional issue date')
     parser.add_argument(
         '-T',
```

Comment:
> On the very next line, you're changing the long-form arg.

Oh, sorry I didn't realize I changed that from designation to title.  I've reverted that change.